package.json

```diff
@@ -76,6 +77,7 @@
     "stylelint": "^13.6.1",
     "stylelint-config-prettier": "^8.0.0",
     "stylelint-config-twbs-bootstrap": "^2.1.0",
+    "typescript": "^4.5.5",
```

Comment:
Typescript is a requirement for this, or...?

Reply:
Typescript is a dependency for `prettier-plugin-organize-imports`.